cptv/reader.py

```diff
@@ -214,13 +214,11 @@ def __iter__(self):
                 last_ffc_time = timedelta(
                     milliseconds=fields.get(Field.LAST_FFC_TIME, 0)
                 )
-            else:
-                time_on = None
-                last_ffc_time = None
-            if self.version >= 3:
                 temp_c = fields.get(Field.TEMP_C, 0)
```

Comment:
I presume that field.get doesn't throw an error if the field is missing?   Because some v2.0 files will still have time_on field, but not the newer fields.

Reply:
nope it's not a problem the test v2.cptv doesn't have those fields